README.md

```diff
@@ -1,15 +1,75 @@
-## My Project
+## amazon-kinesis-video-streams-rtp
 
-TODO: Fill this README out!
+The goal of the Real-time Transport Protocol (RTP) library is to provide
+RTP Serializer and Deserializer functionalities. Along with this, RTP library
+also provide codec packetization and depacketization functionality for G.711,
+VP8, Opus and H.264 codecs.
 
-Be sure to:
+## What is RTP?
 
-* Change the title in this README
-* Edit your repository description on GitHub
+[Real-Time Transport Protocol (RTP)](https://en.wikipedia.org/wiki/Real-time_Transport_Protocol),
+as defined in [RFC 3550](https://datatracker.ietf.org/doc/html/rfc3550), is a data transport protocol
+used to deliver real-time data such as audio or video over IP networks. An RTP packet comprises of a
+header followed by the payload.
 
-## Security
+A RTP packet consist of following fields followed by an optional RTP Header extension.
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+```
+        0                   1                   2                   3
```

Comment:
Can we point to the RFC we referred to for future reference?

Reply:
The RFC [RFC 3550](https://datatracker.ietf.org/doc/html/rfc3550), is mentioned just above this line. Is that fine? Or you are referring to different RFC?

Reply:
Ah. Ok. I missed it then.